### PR TITLE
[Merged by Bors] - feat: port Archive.Imo.Imo1959Q1

### PR DESCRIPTION
**Pull request by urkud**
---

I suggest that we rename all files to use `IMO` instead of `Imo` in
one go after porting them.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by semorrison** [comment]
bors merge

File: Archive/Imo/Imo1959Q1.lean [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Prove that the fraction `(21n+4)/(14n+3)` is irreducible for every natural number `n`.
+
+Since Lean doesn't have a concept of "irreducible fractions" per se, we just formalize this
+as saying the numerator and denominator are relatively prime.
+-/
+
+
+open Nat
+
+namespace IMO1959Q1
```
Comment (semorrison) [inline Archive/Imo/Imo1959Q1.lean:26]:
```suggestion
namespace Imo1959Q1
```
or `Imo1959_q1`, I guess.

We should be consistent with the other PRs, which are presumably going to use the default capitalisation that mathport produces.

We can do a mass cleanup afterwards.

**Comment by semorrison** [comment]
bors d+

**Comment by urkud** [comment]
bors merge

**Pull request closed without merging**